content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -240,10 +398,50 @@ com.datadoghq.tags.version
 ```
 
 As explained in the full configuration, these labels can be set in a Dockerfile or as arguments for launching the container.
+{{< /collapse-content >}} 
+
+[1]: /tracing/services/deployment_tracking/
+[2]: /watchdog/faulty_deployment_detection/
+[3]: /integrations/guide/source-code-integration/?tab=go#embed-git-information-in-your-build-artifacts
 
 {{% /tab %}}
 
 {{% tab "ECS" %}}
+
+#### Set up with automatic version tagging
+##### Full configuration
```

Comment:
`Set up automatic version tagging` is the section but I don't see anything about version below? Is this an oversight?

Reply:
I changed the structure so this doesn't apply anymore